src/popup/locales/en.json

```diff
@@ -307,6 +308,18 @@
         "content": "Transaction data is either broken or Superhero wallet doesn’t support this type of transactions yet. Signing it may cause unexpected results including loss of your funds or other malicious effects.{0}{0}Be careful and sign the data only if you trust the aepp and its owner."
       }
     },
+    "confirm-unsafe-sign": {
+      "heading": "would like you to sign a data.",
+      "superheroChatJwtSign": "would like you to sign a message in order to login.",
+      "title": "Unsafe sign",
+      "typeJwt": "Json Web Token (JWT)",
```

Comment:
Json is usually all caps "JSON" as its an abbreviation https://en.wikipedia.org/wiki/JSON